pkgs/development/tools/casperjs/default.nix

```diff
@@ -28,9 +28,9 @@ in stdenv.mkDerivation rec {
     make test
   '';
 
-
   installPhase = ''
-    mv $PWD $out
+    mkdir $out
```

Comment:
`cp` is almost always used; I haven't looked too closely at this, but it looks like you're copying stuff from the build directory into `$out`; in that case, `cp` will not create extra copies, as the build directory is not retained.

In the end it doesn't matter that much, but what you're doing here is a bit odd (compared to what is normally done).